site/_posts/2017-04-17-Fine-tuning-Fine.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Last Week in ManageIQ: And Finally . . . Fine-tuning Fine"
+author: AparnaKarve
+date: 2017-04-18
+comments: true
+published: true
+tags: LWIMIQ
+---
+## And Finally . . . Fine-tuning **`Fine`**
```

Comment:
I think that this heading when published will appear directly below the title specified in your config above (which is largely the same) - is that intended?